src/diagnostic-channel-publishers/package.json

```diff
@@ -4,16 +4,15 @@
   "main": ".dist/src/index.js",
   "types": ".dist/src/index.d.ts",
   "scripts": {
-    "build": "tsc --outDir ./.dist && node copyTestAssets.js",
-    "lint": "tslint -c tslint.json -p tsconfig.json",
+    "build": "node node_modules/typescript/bin/tsc --outDir ./.dist && node copyTestAssets.js",
```

Comment:
So I was getting errors from travis that it didn't know about these commands.  I think it's because mocha/tsc/tslint aren't installed globally on the travis machines, but we all have them installed globally when we're doing dev work in this repo.   

Now, I could have travis install these things globally, and then stuff should work, but it seems more preferable to rely on locally installed things vs. globally installed things.